### DISCUSSION TITLE
有和pyspark或者pyflink做融合计划吗？

### DISCUSSION THREAD

**Discussion by KONEONE** [alibaba/GraphScope#3678] [Q&A] [answered]
您好，我目前正在使用BigDL将spark和tensorflow&pytorch做一站式数据开发。想问下，对于GraphScope项目有这方面的计划吗？

**Comment by wenyuanyu** [comment, marked as answer]
Hi @KONEONE, at this moment, we do not have a plan to integrate with pyspark and pyflink. Feel free to share your use cases with us. And see if there is anything we can help.
